picard/ui/coverartbox.py

```diff
@@ -255,28 +283,29 @@ def load_remote_image(self, url, mime, data):
         try:
             coverartimage = CoverArtImage(
                 url=url.toString(),
+                types=[u'front'],
                 data=data
             )
         except CoverArtImageError as e:
             log.warning("Can't load image: %s" % unicode(e))
             return
         if isinstance(self.item, Album):
             album = self.item
-            album.metadata.append_image(coverartimage)
+            album.metadata.set_front_image(coverartimage)
```

Comment:
Can you explain this ?

I'm low on coffee, but it seems to `set_front_image()` is actually **removing** other images having is_front_image set to True. How does this work when a release has multiple front images (currently only possible with CAA provider).